src/+otp/+campbellmooredae/jac.m

```diff
@@ -0,0 +1,5 @@
0./2.0).*2.0,y(1,:).*y(2,:).*y(7,:).*1.0./(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*2.0e+1,0.0,y(1,:).*2.0-y(1,:).*1.0./sqrt(y(1,:).^2+y(2,:).^2).*2.0e+1,1.0./(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*(y(4,:).*(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*2.0-y(2,:).^2.*y(4,:).*2.0e+1+y(1,:).*y(2,:).*y(5,:).*2.0e+1),1.0./(y(1,:).^2+y(2,:).^2).^(5.0./2.0).*(y(1,:).^3.*y(5,:).^2.*2.0e+1-y(2,:).^4.*y(8,:).*2.0e+1+y(1,:).^4.*y(8,:).*sqrt(y(1,:).^2+y(2,:).^2).*2.0+y(2,:).^4.*y(8,:).*sqrt(y(1,:).^2+y(2,:).^2).*2.0+y(2,:).^3.*y(4,:).*y(5,:).*4.0e+1+y(1,:).*y(2,:).^3.*y(9,:).*2.0e+1+y(1,:).^3.*y(2,:).*y(9,:).*2.0e+1+y(1,:).*y(2,:).^2.*y(4,:).^2.*6.0e+1-y(1,:).*y(2,:).^2.*y(5,:).^2.*4.0e+1-y(1,:).^2.*y(2,:).^2.*y(8,:).*2.0e+1-y(1,:).^2.*y(2,:).*y(4,:).*y(5,:).*8.0e+1+y(1,:).^2.*y(2,:).^2.*y(8,:).*sqrt(y(1,:).^2+y(2,:).^2).*4.0),0.0,0.0,0.0,0.0,y(1,:).*y(2,:).*y(7,:).*1.0./(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*2.0e+1,y(7,:).*((y(1,:).^2+y(2,:).^2).^(3.0./2.0)-y(1,:).^2.*1.0e+1).*1.0./(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*2.0,0.0,y(2,:).*2.0-y(2,:).*1.0./sqrt(y(1,:).^2+y(2,:).^2).*2.0e+1,1.0./(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*(y(5,:).*(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*2.0-y(1,:).^2.*y(5,:).*2.0e+1+y(1,:).*y(2,:).*y(4,:).*2.0e+1),1.0./(y(1,:).^2+y(2,:).^2).^(5.0./2.0).*(y(2,:).^3.*y(4,:).^2.*2.0e+1-y(1,:).^4.*y(9,:).*2.0e+1+y(1,:).^4.*y(9,:).*sqrt(y(1,:).^2+y(2,:).^2).*2.0+y(2,:).^4.*y(9,:).*sqrt(y(1,:).^2+y(2,:).^2).*2.0+y(1,:).^3.*y(4,:).*y(5,:).*4.0e+1+y(1,:).*y(2,:).^3.*y(8,:).*2.0e+1+y(1,:).^3.*y(2,:).*y(8,:).*2.0e+1-y(1,:).^2.*y(2,:).*y(4,:).^2.*4.0e+1+y(1,:).^2.*y(2,:).*y(5,:).^2.*6.0e+1-y(1,:).^2.*y(2,:).^2.*y(9,:).*2.0e+1-y(1,:).*y(2,:).^2.*y(4,:).*y(5,:).*8.0e+1+y(1,:).^2.*y(2,:).^2.*y(9,:).*sqrt(y(1,:).^2+y(2,:).^2).*4.0),0.0,0.0,0.0,0.0,-sin(t),cos(t),y(7,:).*2.0-1.0,y(3,:).*2.0,y(6,:).*2.0,y(10,:).*2.0,0.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,y(1,:).*1.0./sqrt(y(1,:).^2+y(2,:).^2).*(sqrt(y(1,:).^2+y(2,:).^2)-1.0e+1).*2.0,1.0./(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*(y(4,:).*(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*2.0-y(2,:).^2.*y(4,:).*2.0e+1+y(1,:).*y(2,:).*y(5,:).*2.0e+1).*2.0,0.0,0.0,-1.0,0.0,-1.0,0.0,0.0,0.0,y(2,:).*1.0./sqrt(y(1,:).^2+y(2,:).^2).*(sqrt(y(1,:).^2+y(2,:).^2)-1.0e+1).*2.0,1.0./(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*(y(5,:).*(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*2.0-y(1,:).^2.*y(5,:).*2.0e+1+y(1,:).*y(2,:).*y(4,:).*2.0e+1).*2.0,0.0,0.0,0.0,0.0,cos(t),sin(t),0.0,0.0,y(3,:).*2.0,y(6,:).*4.0,0.0,0.0,0.0,-1.0,y(1,:).*1.0./sqrt(y(1,:).^2+y(2,:).^2).*(sqrt(y(1,:).^2+y(2,:).^2)-1.0e+1).*2.0,y(2,:).*1.0./sqrt(y(1,:).^2+y(2,:).^2).*(sqrt(y(1,:).^2+y(2,:).^2)-1.0e+1).*2.0,y(3,:).*2.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,-1.0,0.0,0.0,0.0,0.0,y(1,:).*1.0./(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*(-(y(1,:).^2+y(2,:).^2).^(3.0./2.0)+y(1,:).^2.*1.0e+1+y(2,:).^2.*1.0e+1).*-2.0,1.0,0.0,0.0,0.0,0.0,-1.0,0.0,0.0,0.0,y(2,:).*1.0./(y(1,:).^2+y(2,:).^2).^(3.0./2.0).*(-(y(1,:).^2+y(2,:).^2).^(3.0./2.0)+y(1,:).^2.*1.0e+1+y(2,:).^2.*1.0e+1).*-2.0,0.0,0.0,0.0,0.0,0.0,0.0,-1.0,0.0,0.0,y(3,:).*2.0,0.0,1.0,0.0,0.0],[10,10]);
```

Comment:
I can see that this was automatically generated, but it seems like it is in dire need of proper formatting.